src/ekat/util/ekat_feutils.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef EKAT_FEUTILS_HPP
+#define EKAT_FEUTILS_HPP
+
+#include "ekat/ekat.hpp"
+
+#ifdef EKAT_NEEDS_FEENABLEEXCEPT
```

Comment:
If you end up doing the changes re: EKAT_HAVE_FEENABLEEXCEPT, then this ifdef logic needs to be toggled the other way around.